projects/harfbuzz/build.sh

```diff
@@ -17,31 +17,24 @@
 
 # Disable:
 # 1. UBSan vptr since target built with -fno-rtti.
-export CFLAGS="$CFLAGS -fno-sanitize=vptr"
-export CXXFLAGS="$CXXFLAGS -fno-sanitize=vptr"
+export CFLAGS="$CFLAGS -fno-sanitize=vptr -DHB_NO_VISIBILITY"
+export CXXFLAGS="$CXXFLAGS -fno-sanitize=vptr -DHB_NO_VISIBILITY"
 
-# Build the library.
-./autogen.sh
-./configure --enable-static --disable-shared
-make clean
-make -j$(nproc) CPPFLAGS="-DHB_NO_VISIBILITY" V=1 all
-
-# Build the fuzzer.
-$CXX $CXXFLAGS -std=c++11 -Isrc \
-    ./test/fuzzing/hb-shape-fuzzer.cc -o $OUT/hb-shape-fuzzer \
-    $LIB_FUZZING_ENGINE ./src/.libs/libharfbuzz.a
+# setup
+build=$WORK/build
 
-$CXX $CXXFLAGS -std=c++11 -Isrc \
-    ./test/fuzzing/hb-draw-fuzzer.cc -o $OUT/hb-draw-fuzzer \
-    $LIB_FUZZING_ENGINE ./src/.libs/libharfbuzz.a
+# cleanup
+rm -rf $build
+mkdir -p $build
 
-$CXX $CXXFLAGS -std=c++11 -Isrc \
-    ./test/fuzzing/hb-subset-fuzzer.cc -o $OUT/hb-subset-fuzzer \
-    $LIB_FUZZING_ENGINE ./src/.libs/libharfbuzz-subset.a ./src/.libs/libharfbuzz.a
+# Build the library.
+meson --default-library=static -Dfuzzer_ldflags=$LIB_FUZZING_ENGINE \
```

Comment:
if you don't find a solution for getting rid of the single quotes, you can do something like

```
-Dfuzzer_ldflags="$(echo $LIB_FUZZING_ENGINE)"
```

Reply:
Oh, let me try it.

Reply:
now turned to https://github.com/google/oss-fuzz/pull/4076/checks?check_run_id=847216709#step:7:525 which hmm, makes sense somehow.

Reply:
will add a split https://github.com/harfbuzz/harfbuzz/blob/master/test/fuzzing/meson.build#L17 here, that should help I guess

Reply:
https://github.com/harfbuzz/harfbuzz/commit/e04050e3b856d275779a2d010fc59ff230a8c4d8 let's see

Reply:
worked very well, thank you very much :) it makes sense also, we should apply that for ots also.

Reply:
> if you don't find a solution for getting rid of the single quotes

It makes sense I believe (or at least nothing we can in harfbuzz can improve AFAIK), guess meson likes to help on things may become wildcard by quoting them.

Reply:
https://mesonbuild.com/FAQ.html#why-cant-i-specify-target-files-with-a-wildcard